data/monarchs.json

```diff
@@ -1,10 +1,10 @@
 [
-{"name":"Elizabeth","start":1565,"end":1603,"index":0},
+{"name":"Elizabeth I","start":1565,"end":1603,"index":0},
```

Comment:
When this dataset was made, only Elisabeth was needed to fully qualify who is meant.